service.partymode.autostart/addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="service.partymode.autostart"
        name="Partymode autostart"
-       version="0.8.8b"
+       version="0.9.0b"
        provider-name="muttley">
```

Comment:
Shouldn't this be your name? :)

Reply:
ehhh, you have right but "muttley" is taken on github :(

Reply:
Ah, so muttley is your forum handle?

Reply:
more or less :) 
If you prefer next time I change provider-name to "elbowz"